package.json

```diff
@@ -67,6 +67,7 @@
     "babel-jest": "24.9.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-istanbul": "6.0.0",
+    "check-prop-types": "^1.1.2",
```

Comment:
TODO: discuss with the team: 
Seems like a helpful idea to verify prop-types on PUBLIC libraries.